internal/stickysessions/endpoints.go

```diff
@@ -86,7 +86,8 @@ func NewEndpointStoreFromEndpointItems(input []EndpointStoreItem, includeNonRead
 func NewEndpointStoreFromEndpointSlices(input []*discoveryV1.EndpointSlice, containerPortName string) *EndpointStore {
 	items := []EndpointStoreItem{}
 	for _, endpointSlice := range input {
-		items = append(items, NewEndpointStoreItems(endpointSlice, containerPortName)...)
+		es := endpointSlice
```

Comment:
should add a comment that this is to deal with the weird go loop behavior so it doesn't get accidentally deleted in the future.

Reply:
Added.